telegram/_bot.py

```diff
@@ -459,6 +461,9 @@ async def _send_message(
         if reply_markup is not None:
             data["reply_markup"] = reply_markup
 
+        if message_thread_id is not None:
+            data["message_thread_id"] = message_thread_id
+
```

Comment:
> Is it just to avoid repetition in the methods? 

jup, that's the use that I see in `_send_message`. 

> If so, we should add a comment here and also maybe add caption_entities.

sure :) maybe PRing makes sense after this is merged :D should we open an issue in the meantime?